include/novas.h

```diff
@@ -19,11 +19,15 @@
 #ifndef _NOVAS_
 #define _NOVAS_
 
-#include <math.h>   // for M_PI
+#include <math.h>   // for sin, cos
 #include <stdlib.h> // NULL
 #include <stdint.h>
 #include <time.h>
 
+#ifndef M_PI
+#    define M_PI 3.14159265358979323846
```

Comment:
You'll need to add a doxygen comment above the #define, e.g.

/// Definition of &pi; in case it's not defined in math.h